create-chroot.sh

```diff
@@ -11,6 +11,12 @@ then
     exit 1
 fi
 
+if [ "${HOMEWORLD_CHROOT}" =~ " " ]
```

Comment:
Do you mean $(basename "${HOMEWORLD_CHROOT}")? It's fine if there are spaces in ancestor directories.

Reply:
It's probably a bad thing for there to be spaces in ancestor directories.

Might it work? Yeah.

But spaces tend to wreck everything, so this is safer.

Reply:
I don't think we should be setting constraints on the user's wider system, but this matter is minor enough that I won't let it block the PR.

Reply:
"don't use spaces" is a fairly normal constraint on a user's system.